docs/settings/alert-action-settings.asciidoc

```diff
@@ -58,3 +58,16 @@ You can configure the following settings in the `kibana.yml` file.
 ==== Alert settings
 
 You do not need to configure any additional settings to use alerting in {kib}.
+
+[float]
+[[event-log-ilm]]
+==== Event log index lifecycle managment
+
+Alerts and actions log their activity in a set of "event log" indices.  These indices are configured with an index lifecycle managment (ILM) policy which can be customized.  The default policy will perform a rollover of the index when it reaches 50GB in size, or after 30 days.  Indices over 90 days old will be deleted.
+
+The name of the index policy is `kibana-event-log-policy`.  It will be created when Kibana starts, if it doesn't exist, otherwise it will never be modified by Kibana.  It can be customized for your environment.
```

Comment:
```suggestion
The name of the index policy is `kibana-event-log-policy`.  {kib} creates the index policy on startup, if it doesn't already exist.  You can customize the index policy prior to startup, but not after.
```

Reply:
I didn't understand the part about "otherwise it will never be modified by Kibana" so my last sentence is a guess.

Reply:
The idea is that we create the ILM policy if it doesn't already exist, with some default values.  But we never modify it after creating it - there is no UI in Kibana to explicitly modify it (outside of the general ui for all ILM policies that Kibana has).  We also never programmatically modify it.

So a customer can make modifications to the policy, to keep data for a shorter or longer period of time, etc.  And not worry that alerting will somehow "overwrite" their settings while it's running, or when upgrading versions, etc.  We just create it when needed, and then never directly access it (even for reading!) after that.

Reply:
So the wording "You can customize the index policy prior to startup, but not after." is exactly wrong! hehe

Customers should **NOT** "pre-create" the index policy, because then we won't create one, but we actually DO need to create it to set all the index patterns, aliases, etc.  And customers may modify the ILM policy at any time after it's created.  Note again, the policy is created at Kibana startup time; this was added in ~7.9 time frame, so anyone upgrading from 7.9 or higher will already have one.  Brand new deployments will have one created when Kibana starts up.

Reply:
new suggested wording from @gchaps:

> Kibana creates the index policy on startup, if it doesn't already exist.  The index policy can be customized for your environment, but Kibana never modifies the index policy after creating it.

Reply:
I updated the suggestion.